Packages/vcs/Lib/vtk_ui/manager.py

```diff
@@ -78,6 +78,7 @@ def detach(self):
             w.detach()
         if self.window.HasRenderer(self.renderer):
             self.window.RemoveRenderer(self.renderer)
+        self.renderer.RemoveAllViewProps()
```

Comment:
You shouldn't need this but if you do then you want to set self.renderer = None since at this point I think it won't be of further use.